client/src/_services/account.service.js

```diff
@@ -0,0 +1,48 @@
+import config from 'config';
+
+export const accountRepository = {
+    create,
+    get,
+    upgradePlan
+};
+
+function create() {
+    const requestOptions = {
+        method: 'POST',
+        withCredentials: true,
+        headers: { 'Content-Type': 'application/json',
+                   'Accept': 'application/json',
+                   'Authorization': `Bearer ${sessionStorage.getItem("Token")}` },
+        body: JSON.stringify({ "plan": 1 })
+    };
+    
+
+    return fetch(`${config.apiUrl}/api/account`, requestOptions)
+        .then(response => response.json())
+}
+
+function upgradePlan(id) {
+    const requestOptions = {
+        method: 'PUT',
+        withCredentials: true,
+        headers: { 'Content-Type': 'application/json',
+                   'Accept': 'application/json',
+                   'Authorization': `Bearer ${sessionStorage.getItem("Token")}` },
```

Comment:
a lot of code duplication around handling security headers and API calls in general.